Application/EdFi.Ods.Api.IntegrationTestHarness/Program.cs

```diff
@@ -29,31 +29,15 @@ public static async Task Main(string[] args)
                 .ConfigureAppConfiguration(
                     (hostingContext, config) =>
                     {
-                        var env = hostingContext.HostingEnvironment;
-
                         config
                             .SetBasePath(Directory.GetParent(AppContext.BaseDirectory).FullName)
                             .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
-                            .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true, reloadOnChange: true)
-                            .AddJsonFile("appsettings.user.json", optional: true, reloadOnChange: true);
-
+                            .AddJsonFile($"appsettings.{hostingContext.HostingEnvironment.EnvironmentName}.json", optional: true, reloadOnChange: true);
 
-                        if (env.IsDevelopment() && !string.IsNullOrEmpty(env.ApplicationName))
+                        if (hostingContext.HostingEnvironment.IsDevelopment())
                         {
-                            var appAssembly = Assembly.Load(new AssemblyName(env.ApplicationName));
-                            if (appAssembly != null)
-                            {
-                                config.AddUserSecrets(appAssembly, optional: true);
-                            }
+                            config.AddUserSecrets<Program>();
                         }
-
-                        config.AddEnvironmentVariables();
-
-                        if (args != null)
-                        {
-                            config.AddCommandLine(args);
-                        }
-
                     })
```

Comment:
is something preventing this one from being removed? I know we've had trouble with it copying the json files over to the output folder but I thought we had fixed that.

Reply:
When I removed ConfigureAppConfiguration completely , Mode value is passed empty for TestHarness . So  I added this code to fix that .

Reply:
I think you still need .SetBasePath(Directory.GetParent(AppContext.BaseDirectory).FullName) for it to pick up the config files, similar to what @jamessmckay  had raised earlier.  Shouldn't need the rest.

Reply:
It looks like the default builder will pull from the appsettings.json in the project root by default (not the bin/ folder). When published it will pull from the appsettings.json directly in the package. should we just use the default behavior here?

Reply:
Oh that's awesome, so it works like web sites did.  Let's make sure everything aligns with that then including the Powershell expectations.

Reply:
As per Ben's suggestion  I just have .SetBasePath(Directory.GetParent(AppContext.BaseDirectory).FullName) to pick up the config files now